lib/crawly/worker.ex

```diff
@@ -158,8 +158,8 @@ defmodule Crawly.Worker do
              parsed_item: Crawly.ParsedItem.t(),
              result: {:ok, :done}
   defp process_parsed_item({parsed_item, response, spider_name}) do
-    requests = Map.get(parsed_item, :requests, [])
-    items = Map.get(parsed_item, :items, [])
+    requests = Map.get(parsed_item, :requests) || []
```

Comment:
Sorry, I don't quite understand the difference between the:
`requests = Map.get(parsed_item, :requests) || []` and 
`requests = Map.get(parsed_item, :requests, [])`

Reply:
```elixir
iex(1)> Map.get(%{requests: nil}, :requests, [])
nil
iex(2)> Map.get(%{requests: nil}, :requests) || []
[]
```

Reply:
Line 164 expects a list and fails on `nil`:

```elixir
iex(1)> Enum.each(nil, &(&1))
** (Protocol.UndefinedError) protocol Enumerable not implemented for nil of type Atom
...
iex(2)> Enum.each([], &(&1))
:ok
```

Reply:
Ah, ok. Clear now